src/App.tsx

```diff
@@ -10,32 +12,28 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabsId, setTabsId] = useState('tab-1');
```

Comment:
Consider renaming the state update function `setTabsId` to `setSelectedTabId` for better clarity and consistency with the `selectedTabId` state variable.